src/shared/components/card/card-content/game-card-content.tsx

```diff
@@ -21,32 +21,36 @@ interface GameCardContentProps {
   teamLeft: TeamProps;
   teamRight?: TeamProps;
   showLivePill?: boolean;
+  onPress?: () => void;
 }
 
 export const GameCardContent = ({
   teamLeft,
   teamRight,
   showLivePill = false,
+  onPress,
 }: GameCardContentProps) => {
   const styles = useStyles(getStyles);
 
   return (
-    <View>
-      <View style={styles.header}>
-        <Typographies.Label color={styles.header.color} verticalTrim>
-          17 SEPT. 2023 · LEC · BO5
-        </Typographies.Label>
-        {showLivePill && (
-          <View style={styles.livePill}>
-            <LivePill />
-          </View>
-        )}
+    <Pressable onPress={onPress}>
```

Comment:
We should use [`TouchableScale`](https://github.com/wanned/karmine-corp-app/blob/main/src/shared/components/touchable-scale/touchable-scale.tsx). (verify that the other cards also use it)